From e8e38492ccb8cbbd386f4499953b6d61a20b9e0f Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Tue, 19 Sep 2023 08:28:14 -0400
Subject: [PATCH] Remove useless shebang lines from package modules

These files do not have the executable bit set in their filesystem
permissions, so a shebang line is not useful. Furthermore, they are not
script-like (no interesting side effects when executed directly) anyway.
---
 helpdev/__init__.py | 1 -
 helpdev/__main__.py | 1 -
 2 files changed, 2 deletions(-)

diff --git a/helpdev/__init__.py b/helpdev/__init__.py
index da79598..e4a9474 100644
--- a/helpdev/__init__.py
+++ b/helpdev/__init__.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
 """HelpDev - Extracts information about the Python environment easily.
diff --git a/helpdev/__main__.py b/helpdev/__main__.py
index c29398b..ed260aa 100644
--- a/helpdev/__main__.py
+++ b/helpdev/__main__.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
 """HelpDev - Extracts information about the Python environment easily.
-- 
GitLab

